### PR TITLE
Tezos example

### PR DESCRIPTION
**Pull request by arch1995**

### CODE REVIEW

File: examples/react-app/src/services/tezosProvider.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      return keyPair;
+    } catch (error) {
+      console.error(error);
+      return null;
+    }
+  }
+
+  const setProvider = async () => {
+    const keyPair = await getTezosKeyPair();
+    // Storing private keys in memory is suitable for development workflows but risky for production use-cases! Use the InMemorySigner appropriately given your risk profile
```
Comment (himanshuchawla009) [inline examples/react-app/src/services/tezosProvider.ts:26]: remove this comment plz, even if they won't sign txn on memory. Web3Auth key is in memory so it doesn't help

Comment (arch1995) [reply examples/react-app/src/services/tezosProvider.ts:26]: ohh ok

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Review by tainguyentt — APPROVED** [review]
lgtm

**Pull request merged**